central/graphql/resolvers/image_components.go

```diff
@@ -704,11 +708,9 @@ func (resolver *imageComponentV2Resolver) TopImageVulnerability(ctx context.Cont
 	// Short path. Full image is embedded when image scan resolver is called.
 	if embeddedComponent := embeddedobjs.ComponentFromContext(resolver.ctx); embeddedComponent != nil {
 		var topVuln *storage.EmbeddedVulnerability
-		var topIndex int
-		for idx, vuln := range embeddedComponent.GetVulns() {
+		for _, vuln := range embeddedComponent.GetVulns() {
 			if topVuln == nil || vuln.GetCvss() > topVuln.GetCvss() {
```

Comment:
I'm not very familiar with this part of the codebase. Let's say we have a CVSS 3.1 score and a CVE 2.0 score, and the 2.0 is higher. We'd prefer that over the 3.1 score?

Reply:
@RTann this is probably best answered by @ksurabhi91 but the best I can tell is this is purely based on what comes in to us from scanner in the CVSS field.  No consideration is taken into account for the NVD score or whether it was NVD 2 vs NVD 3.  

Definitely outside scope of this PR, but also definitely something we should discuss if we have a bug or are not doing it quite right.

Reply:
So we don't lose it.  Further conversation can move to Jira.

https://issues.redhat.com/browse/ROX-29042